packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -203,14 +236,60 @@ class PageTransitionSwitcher extends StatefulWidget {
   /// The child provided to the transitionBuilder may be null.
   final PageTransitionSwitcherTransitionBuilder transitionBuilder;
 
+  /// A function that wraps all of the children that are transitioning out, and
+  /// the [child] that's transitioning in, with a widget that lays all of them
+  /// out. This is called every time this widget is built. The function must not
+  /// return null.
+  ///
+  /// The default is [PageTransitionSwitcherLayoutBuilder] used is
+  /// [defaultLayoutBuilder].
+  ///
+  /// See also:
+  ///
+  ///  * [PageTransitionSwitcherLayoutBuilder] for more information about
+  ///    how a layout builder should function.
+  ///
+  /// The following example shows a [layoutBuilder] that places the new child inside
+  /// a [Column] that sizes its height depending on the heights of active entries.
+  ///
+  /// ```dart
+  /// layoutBuilder: (
```

Comment:
The example should show a full PageTransitionSwitcher instantiation so people can just copy it.